### PR TITLE
Added new model 'text-davinci-003' to OpenAI connector

### PR DESCRIPTION
**Pull request by PowerRobin**
Added the recently added engine `text-davinci-003` to the connector

### CODE REVIEW

File: independent-publisher-connectors/OpenAI/apiDefinition.swagger.json [hunk 1]
```diff
@@ -87,20 +87,25 @@
...
               "text-davinci-002",
               "text-curie-001",
               "text-babbage-001",
               "text-ada-001"
             ],
             "x-ms-enum-values": [
               {
-                "displayName": "DaVinci - most expensive",
+                "displayName": "DaVinci 003 - added in 12/2022, better than 002 at same cost",
+                "value": "text-davinci-002"
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/OpenAI/apiDefinition.swagger.json:105]: @PowerRobin The 'x-ms-enum-values' extension has duplicate values. You can only have one display name mapped to a value. Line 105 and line 109.

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]

File: independent-publisher-connectors/OpenAI/apiDefinition.swagger.json (hunk 1 above)
Comment (PowerRobin) [reply independent-publisher-connectors/OpenAI/apiDefinition.swagger.json]: Oh yes, line 105 has to be "text-davinci-003", my bad.
Fixed it!

**Review by PowerRobin — COMMENTED** [review]
Fixed it

**Comment by Amjed-Ayoub** [comment]
[[certify-connector]]

**Comment by Amjed-Ayoub** [comment]
Hello @PowerRobin,
I hope you are doing well.
Congratulations, your pull request is approved. We will move forward with the certification process. We are preparing your connector for production deployment and will queue up for the next deployment schedule. Once your connector onboards the next deployment schedule, it starts to deploy your connector in our production environments that typically takes 3-4 weeks. 
Please let us know if you have any questions.
Thank you very much for working with us.

**Comment by sriyen-msft** [comment]
@PowerRobin 

Deployment freeze notice: Due to Christmas and Holidays season in the United States, we have temporarily halted deployments of all the connectors till Monday, 2nd January 2023.

Your connector will be queued up for the upcoming deployment schedule which is on Friday 6th January 2023, the production deployment of your connector in all regions will be expected to be completed by Thursday, 26th January 2023.

Please let me know if you have any questions.

Best Regards,
Srikanth Y.

**Pull request closed without merging**